projects/aca-content/src/lib/components/favorite-libraries/favorite-libraries.component.ts

```diff
@@ -47,6 +49,10 @@ export class FavoriteLibrariesComponent extends PageComponent implements OnInit
   list: FavoritePaging;
   isSmallScreen = false;
   columns: DocumentListPresetRef[] = [];
+  searchVisibility = false;
+  isMainActionPresent: boolean;
```

Comment:
you copied `isMainActionPresent` from `CreateMenuComponent` but this feature is not used in this component